Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputView.swift

```diff
@@ -7,23 +7,15 @@ import Storage
 import SwiftUI
```

Comment:
I think after those recent commits, it appears like newly added card doesn't get added to the list of cards. However, navigating back to settings and into the CC autofill page again, you see the card appear.

Adding this comment although it may not relate to changes here exactly. It's so that we can chat in a thread here, as requested.

Reply:
Addressed!